BridgeCareApp/BridgeCare/Services/SummaryReport/BridgeRateDeckAreaWorkSummary.cs

```diff
@@ -0,0 +1,185 @@
+﻿using BridgeCare.Models;
+using OfficeOpenXml;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Web;
+
+namespace BridgeCare.Services
+{
+    public static class BridgeRateDeckAreaWorkSummary
+    {
+        #region constants
+        const string BridgeCare = "Bridge Care";
+        const string Good = "Good";
```

Comment:
This should be an enumeration or a set of classes. Preferably the latter.

Reply:
I used resources file for keeping all these const strings.